src/components/baklava-options-plugin/OptionPlugin.ts

```diff
@@ -0,0 +1,37 @@
+/* eslint-disable */
```

Comment:
Camel case this too please. Also specify the exact rule in eslint-disable. Other than that, looks great.

Reply:
I thought that since it's a class it should be upper camel case?

Reply:
I was mistaken, not for file names.